test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld

```diff
@@ -607,6 +607,297 @@
             "@type": "owl:Class",
             "@id": "knora-api:BooleanValue"
         },
+        {
+            "@id": "knora-api:BundleFileValue",
+            "@type": "owl:Class",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "knora-api:FileValue"
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:arkUrl"
+                    },
+                    "owl:cardinality": 1,
+                    "knora-api:isInherited": true
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:attachedToUser"
+                    },
+                    "owl:cardinality": 1,
+                    "knora-api:isInherited": true
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:deleteComment"
+                    },
+                    "owl:maxCardinality": 1,
+                    "knora-api:isInherited": true
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:deleteDate"
+                    },
+                    "owl:maxCardinality": 1,
+                    "knora-api:isInherited": true
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:deletedBy"
+                    },
+                    "owl:maxCardinality": 1,
+                    "knora-api:isInherited": true
+                },
+                {
+                    "@type": "owl:Restriction",
+                    "owl:onProperty": {
+                        "@id": "knora-api:documentFileValueHasPageCount"
```

Comment:
Shouldn't this be bundleFileValue... instead of documentFileValue...? And ...pageCount seems strange to me for bundle files.